src/TabViewPanResponder.js

```diff
@@ -94,12 +94,8 @@ function forHorizontal(props: Props) {
   }
 
   return {
-    onStartShouldSetPanResponder: (evt: GestureEvent, gestureState: GestureState) => {
-      return canMoveScreen(evt, gestureState);
-    },
-    onStartShouldSetPanResponderCapture: (evt: GestureEvent, gestureState: GestureState) => {
-      return canMoveScreen(evt, gestureState);
-    },
+    onStartShouldSetPanResponder: () => false,
+    onStartShouldSetPanResponderCapture: () => false,
```

Comment:
I changed this to return false as sometimes these callouts will fire before the onMove capture and startingDirection does not get set. 

Additionally there shouldn't be any need for these to ever return true (unless they are there for some edge case reason I have not noticed).